### PR TITLE
[20176] Bump version to v2.13.1

### PR DESCRIPTION
**Pull request by MiguelCompany**
Bump version and release notes

### CODE REVIEW

**Review by cferreiragonz — APPROVED** [review]

File: docs/notes/notes.rst [hunk 1]
```diff
@@ -5,72 +5,23 @@
...
-    18. Fix build with TLS, when `SECURITY=OFF` and `NO_TLS=OFF`.
-
-2. CI fixes:
+This patch release includes the following **fixes**:
 
-    1. Fix colcon on github CI.
-    2. Better handling of trigger events in docs CI.
+1. Revert wrong fix for TCP deadlock on channel reuse.
+2. Fix wrong history selection when removing PDP samples.
+3. Fix data race when processing incoming PDP samples on different threads.
```
Comment (JesusPoderoso) [inline docs/notes/notes.rst:16]: Include latest  merged PRs
```suggestion
3. Fix data race when processing incoming PDP samples on different threads.
4. Fix DNS test filter in CMakeLists
5. Downgrade CMake minimum required version to 3.20
6. Update versions.md
7. Update PR template to include check for PR description, title and backports
8. Update Fast CDR submodule to v2.1.3

```

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]

File: docs/notes/notes.rst (hunk 1 above)
Comment (MiguelCompany) [reply docs/notes/notes.rst]: 666c8ce

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Pull request merged**